comptests/motr/test_motr_di.py

```diff
@@ -210,11 +389,87 @@ def test_m0cp_m0cat_block_corruption_unaligned(self):
         -s 4096 -c 1 -o 1048583 /root/myfile -L 3 -u -O 0
         -o 1048583 -s 4096 -c 10 -L 3 /root/dest_myfile
         """
-        count_list = [['10', '10']]
-        bsize_list = ['4K']
-        layout_ids = ['3']
+        count_list = [["10", "10"]]
+        bsize_list = ["4K"]
+        layout_ids = ["3"]
         offsets = [4096]
+
+        # Todo logging steps
+        self.m0cp_corrupt_data_m0cat(layout_ids, bsize_list, count_list, offsets)
+
+    # @pytest.mark.skip(reason="Feature Unavailable")
+    @pytest.mark.tags("TEST-41768")
+    @pytest.mark.motr_di
+    def test_corrupt_parity_degraded_aligned(self):
+        """
+        Degraded Mode: Parity corruption and detection with M0cp and M0cat
+        Bring the setup in degraded mode by restating m0d with delay
+        and then follow next steps:
+        Copy motr object with m0cp
+        Identify parity block using m0trace logs created during m0cp
+        Corrupt parity block using m0cp+error_injection.py script
+        Read from object with m0cat should throw an error.
+        -s 4096 -c 10 -o 1048583 /root/infile -L 3
+        -s 4096 -c 1 -o 1048583 /root/myfile -L 3 -u -O 0
+        -o 1048583 -s 4096 -c 10 -L 3 /root/dest_myfile
+        """
+        count_list = [["4", "1"]]
```

Comment:
we can send only ['4'] as we are not using cp_update to corrupt.

Reply:
As discussed will take this later as it might need bigger fix for trivial change.